linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/TreeDSL.scala

```diff
@@ -107,6 +107,8 @@ private[emitter] object TreeDSL {
       Assign(self, that)
 
     def prototype(implicit pos: Position): Tree = self DOT "prototype"
+    def length(implicit pos: Position): Tree = self DOT "length"
+    def u(implicit pos: Position): Tree = self DOT "u"
```

Comment:
I think it is OK to introduce these, but we should use them consistently. Which even this commit doesn't (for `length`).